### PR TITLE
registration form posting to DB

### PR DESCRIPTION
**Pull request by CurtCato**
got the registration form posting user input to the DB.

### CODE REVIEW

**Review by mmcdevitt1997 — APPROVED** [review]
👍

File: src/scripts/component/form.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    <button id="loginBtn">Login</button>
+    `
+    loginEvent()
+}
+
+//function to create the registration form
+function createRegistrationForm() {
+    let formContainer = document.querySelector("#container")
+    formContainer.innerHTML = `
+    <h1>Welcome User. Please create user ID</h1>
```
Comment (lindseyemaddox) [inline src/scripts/component/form.js:21]: Could we get super fancy and welcome the user by identifier?

File: src/scripts/component/homepage.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// Curt
+
+function populateHomepage () {
+    let homepageContainer = document.querySelector("#container")
+    homepageContainer.innerHTML = `
+    <div id="mainHomepageDiv">
+    <h1>Bitchin. It's a homepage</h1>
+    <section id="eventsSection></section>
```
Comment (lindseyemaddox) [inline src/scripts/component/homepage.js:8]: Missing some quotation marks here and below

Comment (CurtCato) [reply src/scripts/component/homepage.js:8]: thanks. fixed it

File: src/scripts/component/form.js (hunk 1 above)
Comment (CurtCato) [reply src/scripts/component/form.js:21]: no! too fancy!

**Pull request merged**